semconv/src/main/java/io/opentelemetry/semconv/resource/attributes/package-info.java

```diff
@@ -4,11 +4,11 @@
  */
 
 /**
- * OpenTelemetry semantic attributes.
+ * OpenTelemetry semantic attributes for resources.
  *
  * @see io.opentelemetry.semconv.trace.attributes.SemanticAttributes
```

Comment:
this should point to the resource attributes, yes?

Reply:
Done.